luigi/scheduler.py

```diff
@@ -870,6 +871,7 @@ def _serialize_task(self, task_id, include_deps=True, deps=None):
             'priority': task.priority,
             'resources': task.resources,
             'tracking_url': getattr(task, "tracking_url", None),
+            'status_message': task.status_message
```

Comment:
This line is causing a stack-trace like this:

```
Traceback (most recent call last):
  File "/zserver/Python-3.4.3/lib/python3.4/site-packages/tornado/web.py", line 1443, in _execute
    result = method(*self.path_args, **self.path_kwargs)
  File "/zserver/apps/luigi/code-for-luigid/luigi/server.py", line 101, in get
    result = getattr(self._scheduler, method)(**arguments)
  File "/zserver/apps/luigi/code-for-luigid/luigi/scheduler.py", line 986, in task_list
    serialized = self._serialize_task(task.id, False)
  File "/zserver/apps/luigi/code-for-luigid/luigi/scheduler.py", line 868, in _serialize_task
    'status_message': task.status_message
```

For anyone using an old pickled state-file.